packages/nestjs/src/sdk.ts

```diff
@@ -0,0 +1,16 @@
+import { applySdkMetadata } from '@sentry/core';
+import type { NodeOptions } from '@sentry/node';
+import { init as nodeInit } from '@sentry/node';
+
+/**
+ * Initializes the NestJS SDK
+ */
+export function init(options: NodeOptions | undefined = {}): void {
```

Comment:
h: SDKs are supposed to return the `Client`

Reply:
updated